src/librustc/mir/interpret/value.rs

```diff
@@ -98,23 +93,23 @@ impl<'tcx> Scalar {
     pub fn ptr_null<C: HasDataLayout>(cx: C) -> Self {
         Scalar::Bits {
             bits: 0,
-            defined: cx.data_layout().pointer_size.bits() as u8,
+            size: cx.data_layout().pointer_size.bytes() as u8,
         }
     }
 
+    pub fn to_value_with_len<C: HasDataLayout>(self, len: u64, cx: C) -> Value {
+        ScalarMaybeUndef::Scalar(self).to_value_with_len(len, cx)
+    }
```

Comment:
Why do we have this, but not the vtable version?

Reply:
the vtable one wasn't used anywhere

Reply:
I think in terms of API consistency it still makes sense to have them both.